compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -917,6 +945,7 @@ options numProcessors =
         <> cmdMigrate numProcessors
         <> cmdInit
         <> cmdCompile numProcessors
+        <> cmdLint numProcessors
```

Comment:
Why do we hide this? It seems like a useful command that someone might want to add to their CI setup or something like that.

Reply:
What do you mean by hidden?

Reply:
`cmdCompile` is hidden because users are supposed to use `cmdBuild`. Given that there is no alternative to `cmdLint` I would put it on the same level as `cmdBuild`.

Reply:
Roger roger.

Reply:
✔️